### PR TITLE
Speakers

### PR DESCRIPTION
**Pull request by sntgopl**
-I'll leave the link to the presentation video here: https://www.loom.com/share/d7fe138b051a4b55aa148799586d8194
-Created a pull request for the review of my first capstone project.
-Created index html and css file with both mobile and desktop version.
-Created about html and css file with both mobile and desktop version.
-Created Javascript index file for menu deployment for mobile version.
-Created Speakers js file for index speaker section deployment in mobile version.
-Created pull request for review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
-First Capstone project Microverse
+# Capstone-Project-Module-1
+ First Capstone Project - Microverse
+
+## Built With
+ Github, Visual Studio Code, HTML, CSS & Javascript
+
+## You can find us in:
+ Live Demo:
+
```
Comment (elisha2kyakpo1) [inline README.md]: - [ ] Kindly deploy the app, using GitHub pages, and make the app link available in the readme file. The project should be deployed properly, all pages, links, and images should work as expected.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+				<img src="images/creative-commons-license-logo.jpeg" alt="logo">
+			</section>
+			<section class="registry">
+				<h2>See the past CC Global Summits</h2>
+				<span class="line"></span>
+				<p>Take a look at the last two CC Global Summits which took place in Buenos Aires and in Warsaw.</p>
+				<div>
+					<a class="conferenceImg" href="#"><h4>2019</h4><br><h5>CC Global Summit in Buenos Aires</h5></a>
+					<a class="conferenceImg1" href="#"><h4>2020</h4><br><h5>CC Global Summit in Warsaw</h5></a>
+				</div>
```
Comment (elisha2kyakpo1) [inline about.html:50]: - [ ] Kindly align the images properly on mobile screens, please take a look at the image below. Your pictures overlap the screen.

image-overlap

- [ ] The heading on the picture below is too big and it is overlapping as well.

mobile-logo

File: index.html [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+				<a id="hamburger" href="#"><div></div><div></div><div></div></a>
+				<div class="m-nodisplay menu-bar">
+					<a class="menu-option" href="about.html">About us</a>
+					<a class="menu-option" href="#">News</a>
+					<a class="menu-option" href="#">Program</a>
+					<a class="menu-option" href="#">Join</a>
+					<a class="menu-option" href="#">Sponsor</a>
+					<a class="menu-button" href="#">CC Campaing</a>
+				</div>
+    	</nav>
```
Comment (elisha2kyakpo1) [inline index.html:35]: nav-og
- [ ] Kindly add the top nav in black, the project has a small nav with social media icons.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+						<h3 id="name">Lecturer Name</h3>
+						<h4 id="titles">Lercturer`s title</h4>
+						<span class="line2"></span>
+						<p id="description">Lecturer`s description</p>
+					</div>
+				</div>
+				<div class="speaker-card m-nodisplay">
+					<img id="picture" src="images/Picture3.png" alt="speaker-foto">
+					<div id="content">
+						<h3 id="name">Lecturer Name</h3>
```
Comment (elisha2kyakpo1) [inline index.html:85]: - [ ] Let's reduce the space between text and an image, see the screenshots.

### Original
speaker-og

### Yours
speaker-yours

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @sntgopl, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The PR title and description looks professional 👍
- No linter errors 👍
- Good and descriptive commit messages 👍


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @sntgopl ✋



Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**